src/encoded/static/components/experiment.js

```diff
@@ -428,13 +428,23 @@ const ExperimentComponent = ({ context, auditIndicators, auditDetail }, reactCon
         });
     }
 
-    // Collect biosample docs.
+    // Collect biosample docs
+    // Collect plasmid maps from applied_modifictions documents.
```

Comment:
Typo: "applied_modifictions" should be "applied_modifications"
Also, I generally try to be consistent on comments about things like whether there are periods at the ends of lines or not, and put periods at the ends of every line/sentence or none of them